src/locator/MarkdownFileLocator.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs');
+const path = require('path');
+
+const ACCEPTED_EXTENSIONS = ['.md', '.mdwn', '.mdown', '.mkd', '.mkdn', '.markdown', '.Rmd'];
+
+class MarkdownFileLocator {
+    locateFiles(basePath) {
+        if (false === fs.existsSync(basePath)) {
```

Comment:
replacing false === by ```!fs.existsSync(basePath)``` would be better?

Reply:
Added to changes.